src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/PInvokeMarshal.cs

```diff
@@ -269,7 +268,7 @@ public static IntPtr GetCurrentCalleeOpenStaticDelegateFunctionPointer()
         /// <summary>
         /// Retrieves the current delegate that is being called
         /// </summary>
-        public static T GetCurrentCalleeDelegate<T>() where T : class // constraint can't be System.Delegate
+        public static T GetCurrentCalleeDelegate<T>() where T : Delegate
```

Comment:
Do we know why this comment was made and it is now okay to be changed?

Reply:
It should come from a long time ago. I would check git blame.
Seems that I forgot to change it's caller.

Reply:
Oh. This is in native AOT. @MichalStrehovsky Any concerns with this change?

Reply:
Presumably this is just because the code was written before `Delegate` was allowed as a constraint (C# 7.3); https://github.com/dotnet/corert/blame/master/src/System.Private.CoreLib/src/System/Runtime/InteropServices/PInvokeMarshal.cs#L416

Reply:
> Presumably this is just because the code was written before `Delegate` was allowed as a constraint

Yep, pretty sure the author of the comment wanted to say "C# doesn't allow constraint to be Delegate, but T should be a delegate". C# now does allow Delegate constraint. LGTM.